src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/DoNotUseCountWhenAnyCanBeUsedTests.Tests.cs

```diff
@@ -1051,6 +1054,7 @@ End Function
             b = Not Await GetData().AnyAsync()
```

Comment:
That remains as (Await GetData().CountAsync).Equals(IntegerZero)

Reply:
That is not the case for the parentheses scenario `(Await GetData().CountAsync()).Equals(IntegerZero)` or am I missing something?

Reply:
@Jozkee I couldn't get what you mean

Reply:
I was replying to @Evangelink's suggestion; what I got from his comment was that `(Await GetData().CountAsync).Equals(IntegerZero)` (CountAsync without parentheses) should not be caught by the analyzer and I said that I disagree given that the analyzer already catches `(Await GetData().CountAsync()).Equals(IntegerZero)` (CountAsync with parentheses).

We should be consistent.

Reply:
@Jozkee The comment from @Evangelink was before I had fixed the async case. He was pointing to the test failure that was expecting `Not Await GetData().AnyAsync()`, but the actual was `(Await GetData().CountAsync).Equals(IntegerZero)`

Reply:
So, that's what I was missing :)
Thanks for clarifying.